pkgs/applications/video/alass/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, makeWrapper
+, ffmpeg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "alass";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "kaegi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-q1IV9TtmznpR7RO75iN0p16nmTja5ADWqFj58EOPWvU=";
+  };
+
+  cargoSha256 = "sha256-6CVa/ypz37bm/3R0Gi65ovu4SIwWcgVde3Z2W1R16mk=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram "$out/bin/alass-cli" --prefix PATH : "${ffmpeg}/bin"
+  '';
+
+  meta = with lib; {
+    description = "Automatic Language-Agnostic Subtitle Synchronization";
+    homepage = "https://github.com/kaegi/alass";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
refer to https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206

---

Thanks! I already knew about the distinction between Only and Plus. Now I now how to recognise it :)